muikku-elastic-search/src/main/java/fi/otavanopisto/muikku/plugins/search/ElasticSearchProvider.java

```diff
@@ -130,12 +132,28 @@ public SearchResult searchUsers(String text, String[] textFields, Collection<Env
         }
       }
       
+      if (excludeSchoolDataIdentifiers != null) {
+        IdsFilterBuilder excludeIdsFilterBuilder = FilterBuilders.idsFilter("User");
+        for (SchoolDataIdentifier excludeSchoolDataIdentifier : excludeSchoolDataIdentifiers) {
+          excludeIdsFilterBuilder.addIds(String.format("%s/%s", excludeSchoolDataIdentifier.getIdentifier(), excludeSchoolDataIdentifier.getDataSource()));
```

Comment:
is there another (more generic) way to do this without string.format or is it handled like this in the source?